pkg/kapp/logs/pod_container_log.go

```diff
@@ -140,6 +161,15 @@ func (l PodContainerLog) readyToGetLogs() bool {
 	if err != nil {
 		return false
 	}
+	containerStatuses := pod.Status.ContainerStatuses
```

Comment:
what about init containers? this gets run against initContainers + plain containers: https://github.com/vmware-tanzu/carvel-kapp/blob/e0bf1c9d485c5c59a23b4cfe75915782e7fb4b42/pkg/kapp/logs/pod_log.go#L70

Reply:
No, as of today, it doesnt get run against `init containers`. I will add that as well.

Reply:
Adopted